content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -139,4 +161,21 @@ For example: to enable deployments and daemonsets, set
 
 {{< note >}}Individual resource enablement/disablement is only supported in the `extensions/v1beta1` API group for legacy reasons.{{< /note >}}
 
+## Persistence
+
+Kubernetes stores its serialized state in terms of the API resources; the persistence store
+is {{< glossary_tooltip term_id="etcd" >}} although it's possible to change the code to
+implement a different storage mechanism.
+
+{{% /capture %}}
```

Comment:
Should this stuff be under a different heading?

Reply:
Hi @lavalamp 

What heading do you recommend? “Persistence” was what came to mind first.

Reply:
Sorry, I was just looking at the .md, the preview makes it clear that some magic already adds an additional header here for these links.